packages/webapp/__tests__/ProfileIndexPage.tsx

```diff
@@ -102,18 +101,6 @@ const renderComponent = (
   );
 };
 
-it('should show the reading rank history of the user', async () => {
```

Comment:
We are removing the Weekly goals completely, soon. We don't need the test anymore.